### ISSUE TITLE
SEO Settings: Sitemap link should be disabled on non-public sites

### ISSUE THREAD

**Issue by rachelmcr** [Automattic/wp-calypso#5505]
#### Steps to reproduce
1. Starting at URL: https://wpcalypso.wordpress.com/settings/seo
2. Select a site with the site privacy set to hidden or private (not public)
3. Under "XML Sitemap" click the link to the site's sitemap
4. Result: You are taken to a 404 not found error on the site
#### What I expected

The link to the sitemap would be disabled if the site is not public (a sitemap doesn't exist)
#### What happened instead

I was able to click the link and ended up on my site with the message, "Oops! That page can't be found."
#### Browser / OS version

Chrome 50.0.2661.102, Mac OS X 10.11.5
#### Screenshot / Video

![screen shot 2016-05-21 at 12 34 53 pm](https://cloud.githubusercontent.com/assets/8658164/15448135/96cb5e88-1f50-11e6-94bd-bfe71292988c.png)

![screen shot 2016-05-21 at 12 36 17 pm](https://cloud.githubusercontent.com/assets/8658164/15448140/b28cf1fe-1f50-11e6-995f-a904553dde86.png)

**Comment by drw158** [comment]
@roundhill I think the best way to do this might be with this css

```
.is-disabled {
   pointer-events: none;
   cursor: default;
}
```

maybe on the whole `fieldset`

**Comment by roundhill** [comment]
Fixed via #11476
